source/api_cc/src/DeepPotJAX.cc

```
@@ -0,0 +1,581 @@
+  // copy data
+  T result = *data;
+  // deallocate
+  TFE_DeleteOp(op);
+  TF_DeleteTensor(tensor);
+  TFE_DeleteTensorHandle(retval);
+  return result;
+}
+
+template <typename T>
+inline std::vector<T> get_vector(TFE_Context* ctx,
+                                 const std::string func_name,
```

cppcheck

[performance] Function parameter 'func_name' should be passed by const reference.

(passedByValue)